nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/test/java/org/apache/nifi/cluster/coordination/flow/TestPopularVoteFlowElection.java

```diff
@@ -46,6 +45,19 @@
 
 public class TestPopularVoteFlowElection {
 
+    /**
+     * Utility method which accepts {@link NiFiProperties} object but calls {@link StringEncryptor#createEncryptor(String, String, String)} with extracted properties.
```

Comment:
ok yeah this method seems like a good candidate for being in 'nifi-mock' or some nifi test utils module.  This call is needed in a lot of tests.  But we dont want/need additional deps to some module in mainline code as the framework can pull from nifi props as this is doing and avoid another dependency just to share across code and tests..